spec/takeaway_spec.rb

```diff
@@ -0,0 +1,126 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  let(:takeaway) { described_class.new(restaurant) }
+  let(:restaurant) { double() }
```

Comment:
Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments. (https://github.com/bbatsov/ruby-style-guide#no-args-no-parens)